kinetic/distribution.yaml

```diff
@@ -2232,7 +2232,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/RobotWebTools-release/depthcloud_encoder-release.git
-      version: 0.0.5-0
+      version: 0.1.0-0
     source:
```

Comment:
@jihoonl There looks to be the same branch issues here as in melodic: https://github.com/ros/rosdistro/pull/20774/files#r271082812